test/controllers/ApplicationControllerSpec.scala

```diff
@@ -175,7 +176,7 @@ class ApplicationControllerSpec extends BaseSpec with CurrentTaxYear {
       override lazy val getSelfAssessmentServiceResponse = NonFilerSelfAssessmentUser
       override val allowLowConfidenceSA = false
 
-      val r = controller.uplift(Some(ContinueUrl("/personal-account")))(buildFakeRequestWithAuth("GET"))
+      val r = controller.uplift(Some(SafeRedirectUrl("/personal-account")))(buildFakeRequestWithAuth("GET"))
```

Comment:
Can we rename this val to make more readable?

Reply:
We can. And it's been renamed to redirection.